feature_layers/feature-collection-layer-from-portal/src/main/resources/style.css

```diff
@@ -0,0 +1,27 @@
+.panel-region .label {
```

Comment:
Going forward we're trying to limit this CSS file to only the rules that are being used. Once you've changed the sample to only only live in one file, just check which of these you are using, and delete the rest.